koji-containerbuild.spec

```diff
@@ -7,7 +7,7 @@
 
 Name:           koji-containerbuild
 Version:        0.7.4
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Why is this needed?

Reply:
Not sure if its required, as spec was changed, but no actual release was made. I'll keep this in case a new fix release will be done

Reply:
this is always 1 in atomic-reactor and osbs-client, do we really wanna change this?